docs/08-models-and-migrations.md

```diff
@@ -565,11 +662,12 @@ class PostTest extends TestCase
 {
   // other tests...
 
+  /** @test */
```

Comment:
```suggestion
```

Reply:
I think this is related to the duplicated annotation, it was solved in another suggestion.